geode-for-redis/src/test/java/org/apache/geode/redis/internal/data/RedisSortedSetTest.java

```diff
@@ -153,6 +157,58 @@ public void zadd_stores_delta_that_is_stable() {
     assertThat(sortedSet1.hasDelta()).isFalse();
   }
 
+  @Test
+  public void zstoreoperations_stores_delta_that_is_stable() {
```

Comment:
This test  and the one below it should probably be renamed "sortedSetOpStoreResult_*" as there is no method named "zstoreoperations".